packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1836,23 +1836,29 @@ export default class Meeting extends StatelessWebexPlugin {
 
     LoggerProxy.logger.info(`${LOG_HEADER} Muting Audio`);
 
-    return this.audio.toggle({
+    return logRequest(this.audio.toggle({
       mute: true,
       self: true
     })
       .then(() => {
-        LoggerProxy.logger.info(`${LOG_HEADER} Audio mute successful.`);
+        LoggerProxy.logger.info(`${LOG_HEADER} Audio mute was successful.`);
+
         Metrics.postEvent({
           event: eventType.MUTED,
           meeting: this,
           data: {trigger: trigger.USER_INTERACTION, mediaType: mediaType.AUDIO}
         });
-
-        return Promise.resolve();
       })
       .catch((e) => {
         LoggerProxy.logger.error(`${LOG_HEADER} Audio mute error ${e}`);
-      });
+
+        throw new Error(e);
```

Comment:
This method, in order to work with `logRequest()`, requires an error to be thrown in order to properly operate.

Reply:
That's why I didn't use `logRequest`. I didn't want to change the behavior of the method.